### PR TITLE
Add Tips to the Progress Dialog

### PR DESCRIPTION
**Pull request by cjs8487**
Adds a set of tips to the randomization progress dialog that cycle through over time. Looking for more tips to add to the list prior to merging

### CODE REVIEW

File: gui/progressdialog.py [hunk 1]
```diff
@@ -1,16 +1,54 @@
...
-from PySide6.QtWidgets import QProgressDialog
+import random
+from PySide6.QtCore import Qt, QTimer
+from PySide6.QtWidgets import QProgressDialog, QLabel
+
+tips = [
+    "The Bomb Bag and the Hook Beetle are often interchangable. In places you would normally be expected to use the Bomb Bag, look for nearby bomb flowers to grab with the Hook Beetle.",
+    "Think you're stuck somewhere with no way to get back to safety? Look around for a bomb flower. Dying will safely return you to the last statue you saved at or the last loading zone you crossed",
+    "Logic assumes that you will use keys in the worst way possible, so it is impossible to lock youself out of access to keys.",
+    "Enabling tricks is a fun way to expand the logical possiblities of the randomizer, but some tricks can greatly incease the difficulty of seeds",
```
Comment (YourAverageLink) [inline gui/progressdialog.py:9]:
```suggestion
    "Enabling tricks is a fun way to expand the logical possiblities of the randomizer, but some tricks can greatly incease the difficulty of seeds.",
    "Fi can give you some helpful information about the seed, such as which dungeons are required and how many keys you have. You can always call her with D-pad down, even without a sword.",
    "Hold down the B button to quickly advance text.",
```

File: gui/progressdialog.py [hunk 2]
```diff
@@ -1,16 +1,54 @@
-from PySide6.QtCore import Qt
-from PySide6.QtWidgets import QProgressDialog
+import random
+from PySide6.QtCore import Qt, QTimer
+from PySide6.QtWidgets import QProgressDialog, QLabel
+
+tips = [
+    "The Bomb Bag and the Hook Beetle are often interchangable. In places you would normally be expected to use the Bomb Bag, look for nearby bomb flowers to grab with the Hook Beetle.",
+    "Think you're stuck somewhere with no way to get back to safety? Look around for a bomb flower. Dying will safely return you to the last statue you saved at or the last loading zone you crossed",
```
Comment (YourAverageLink) [inline gui/progressdialog.py:7]:
```suggestion
    "Think you're stuck somewhere with no way to get back to safety? Look around for a bomb flower. Dying will safely return you to the last statue you saved at or the last loading zone you crossed.",
```

**Review by YourAverageLink — CHANGES_REQUESTED** [review]
Just adding a few more that I think would be useful.

File: gui/progressdialog.py [hunk 3]
```diff
@@ -1,16 +1,56 @@
-from PySide6.QtCore import Qt
-from PySide6.QtWidgets import QProgressDialog
+import random
+from PySide6.QtCore import Qt, QTimer
+from PySide6.QtWidgets import QProgressDialog, QLabel
+
+tips = [
+    "The Bomb Bag and the Hook Beetle are often interchangable. In places you would normally be expected to use the Bomb Bag, look for nearby bomb flowers to grab with the Hook Beetle.",
+    "Think you're stuck somewhere with no way to get back to safety? Look around for a bomb flower. Dying will safely return you to the last statue you saved at or the last loading zone you crossed,",
```
Comment (YourAverageLink) [inline gui/progressdialog.py:7]:
```suggestion
    "Think you're stuck somewhere with no way to get back to safety? Look around for a bomb flower. Dying will safely return you to the last statue you saved at or the last loading zone you crossed.",
```

Comment (YourAverageLink) [reply gui/progressdialog.py:7]: Sorry i probably typo'd that comma

Comment (cjs8487) [reply gui/progressdialog.py:7]: Nah that was me lmfao. I'l make the change but I'm not gonna push quite yet

**Review by lepelog — APPROVED** [review]
If the tip is wider than 400px the dialog appears in the top left corner of the screen for me, but that's probably just a linux thing, looks good otherwise.

The tips themselves look good.

**Review by lepelog — APPROVED** [review]

**Pull request merged**